idd/Energy+.idd.in

```diff
@@ -448,19 +448,30 @@ PerformancePrecisionTradeoffs,
       \memo This object enables users to choose certain options that speed up EnergyPlus simulation,
       \memo but may lead to small decreases in accuracy of results.
   A1, \field Use Coil Direct Solutions
-      \note If Yes, an analytical or empirical solution will be used to replace iterations in
+      \note If Yes, an analytical or empirical solution will be used to replace iterations in 
       \note the coil performance calculations.
       \type choice
       \key Yes
       \key No
-      \default No
-  A2; \field Zone Radiant Exchange Algorithm
+  A2, \field Zone Radiant Exchange Algorithm
       \note Determines which algorithm will be used to solve long wave radiant exchange among surfaces within a zone.
       \type choice
       \key ScriptF
       \key CarrollMRT
       \default ScriptF
-
+      \default No
```

Comment:
It looks like maybe this `\default No` was meant to stay on A1?  CI was clearly complaining about the duplicated default, and the results are all red.  

We are essentially at IO freeze.  Is there a push to get this in right now?  If so, it will need some attention.

Reply:
Thanks. I saw everything was failing but I didn't understand that it was the double default. Thanks for spotting that. 

I was hoping to squeeze this in prior to IO freeze since it is mostly simple stuff. What is the deadline?